aiida/manage/manager.py

```diff
@@ -86,8 +86,9 @@ def _load_backend(self, schema_check=True):
 
         # Do NOT reload the backend environment if already loaded, simply reload the backend instance after
         if configuration.BACKEND_UUID is None:
-            manager = self.get_backend_manager()
-            manager.load_backend_environment(profile, validate_schema=schema_check)
+            from aiida.backends import get_backend_manager
+            backend_manager = get_backend_manager(self.get_profile().database_backend)
```

Comment:
this is safer?

Reply:
no, was simply necessary to prevent infinite recursion (see other comment)